### ISSUE TITLE
OSX M1, .NET Core 6 - Unable to load shared library 'libHarfBuzzSharp'

### ISSUE THREAD

**Issue by jefim** [mono/SkiaSharp#1759]
When trying to use any HardBuzzSharp (version 2.6.1.8) I am getting the following at runtime on my MacBook M1:

```
Unhandled exception. System.DllNotFoundException: Unable to load shared library 'libHarfBuzzSharp' or one of its dependencies. In order to help diagnose loading problems, consider setting the DYLD_PRINT_LIBRARIES environment variable: dlopen(liblibHarfBuzzSharp, 1): image not found
   at HarfBuzzSharp.HarfBuzzApi.hb_blob_create(Void* data, UInt32 length, MemoryMode mode, Void* user_data, DestroyProxyDelegate destroy)
   at HarfBuzzSharp.Blob.Create(IntPtr data, Int32 length, MemoryMode mode, ReleaseDelegate releaseProc) in HarfBuzzSharp.dll:token 0x600005f+0x10
   at HarfBuzzSharp.Blob..ctor(IntPtr data, Int32 length, MemoryMode mode, ReleaseDelegate releaseDelegate) in HarfBuzzSharp.dll:token 0x6000054+0x0
   at HarfBuzzSharp.Blob..ctor(IntPtr data, Int32 length, MemoryMode mode) in HarfBuzzSharp.dll:token 0x6000053+0x0
   at ConsoleApp1.Program.Main(String[] args) in /.../ConsoleApp1/ConsoleApp1/Program.cs:line 32
```

The same problem exists in e.g. SkiaSharp 2.80.3, but all works fine in SkiaSharp 2.88.0-preview61. The dlopen(liblibHarfBuzzSharp, 1) part looks fishy, but I have no idea what it could indicate. The lib itself is in the correct place (runtimes/osx/native/libHarfBuzzSharp.dylib).

My test program really does not do much, this is the code that crashes:

`var face = new HarfBuzzSharp.Face(new Blob(IntPtr.Zero, 0, MemoryMode.Duplicate), 0);`

Am I doing something wrong or is this a known issue? Any pointers appreciated!

**Comment by Gillibald** [comment]
2.80.3 has no native bits compiled for that arch so it is expected not to work properly

**Comment by jefim** [comment]
Is there anything I can do about it? Or is there any schedules or open issues that I can attempt to tackle for resolve this?

**Comment by mattleibow** [comment]
Let me have a look. I may not have pushed an update to harfbuzzsharp with the new architecture.

**Comment by jefim** [comment]
Hello, I have actually resolved this yesterday as I realized with help from others that I should be referencing SkiaSharp.HarfBuzz rather that directly HarfBuzz package. The SkiaSharp one actually does have this arch builtin and things are working. Thanks for the help.

**Comment by Gillibald** [comment]
If that is true there is something totally wrong. SkiaSharp.HarfBuzz only provides extension methods for HarfBuzzSharp and has no native assets.

**Comment by mattleibow** [comment]
I think I see what happened. I released SkiaSharp v2.80 and HarfBuzzSharp v2.6.1.8. This is the old combo. But, when 2.88 is coming around, the version that goes with that is HarfBuzzSharp v2.6.1.8-preview. I did not bump the HarfBuzzSharp version.

I see a 2.6.1.9 on the preview feed.

Basically, I forgot to bump harfbuzz when I released the preview, and when I did bump, it is still pending a release. I am waiting for the P7 of .NET 6 and .NET Maui.

Pulling in SkiaSharp.HarfBuzz 2.88 actually pulls in an "older" version, but is newer.
